exercises/difference-of-squares/difference_of_squares.py

```diff
@@ -1,10 +1,10 @@
-def square_of_sum():
+def square_of_sum(square_of_sum):
     pass
 
 
-def sum_of_squares():
+def sum_of_squares(sum_of_squares):
     pass
 
 
-def difference():
+def difference(square_of_sum, sum_of_squares):
```

Comment:
`difference` takes only one parameter according to https://github.com/exercism/python/blob/master/exercises/difference-of-squares/difference_of_squares_test.py